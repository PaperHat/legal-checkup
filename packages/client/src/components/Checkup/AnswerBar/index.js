import React from 'react'

import { NO, NOT_SURE, YES } from '../../../state/constants'

import Bar from './Bar'
import ResponseButton from './ResponseButton'

export default function AnswerBar () {
  return (
    <Bar>
      <ResponseButton type={YES} />
      <ResponseButton type={NO} />
      <ResponseButton type={NOT_SURE} />
    </Bar>
  )
}
