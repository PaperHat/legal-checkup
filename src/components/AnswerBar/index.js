import ConnectedNoButton from '../../containers/NoButton'
import ConnectedNotSureButton from '../../containers/NotSureButton'
import ConnectedYesButton from '../../containers/YesButton'
import React from 'react'
import StyledAnswerBar from '../styled/AnswerBar'

export default function AnswerBar () {
  return (
    <StyledAnswerBar>
      <ConnectedYesButton />
      <ConnectedNoButton />
      <ConnectedNotSureButton />
    </StyledAnswerBar>
  )
}
