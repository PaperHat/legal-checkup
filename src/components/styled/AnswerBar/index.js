import styled from 'styled-components'

const AnswerBar = styled.section`
  margin: auto;
  margin-bottom: 41px;
  width: 90%;
  width: 292px;
`
AnswerBar.displayName = 'StyledAnswerBar'

export default AnswerBar
