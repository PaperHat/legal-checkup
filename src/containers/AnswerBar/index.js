import { NO, NOT_SURE, YES } from '@state/constants'

import React from 'react'
import ResponseButton from '../ResponseButton'
import StyledAnswerBar from '@components/styled/AnswerBar'

export default function AnswerBar () {
  return (
    <StyledAnswerBar>
      <ResponseButton type={YES} />
      <ResponseButton type={NO} />
      <ResponseButton type={NOT_SURE} />
    </StyledAnswerBar>
  )
}
