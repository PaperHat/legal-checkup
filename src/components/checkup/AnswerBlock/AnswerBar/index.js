import styled from 'styled-components'
import styledMap from 'styled-map'

import { DESKTOP, MOBILE, TABLET } from '../../../../constants'

const AnswerBar = styled.section`
  margin: auto;
  text-align: center;
  background-color: #ffffff;
  box-shadow: 0 3px 3px 0 rgba(0, 0, 0, 0.2);
  ${styledMap('format', {
    [DESKTOP]: `
    width: 738px;
    padding-bottom:61px;`,
    [MOBILE]: `
    width: 325px;`,
    [TABLET]: `
    width: 708px;
    padding-bottom:61px;`
  })}
`
AnswerBar.displayName = 'AnswerBar'

export default AnswerBar
