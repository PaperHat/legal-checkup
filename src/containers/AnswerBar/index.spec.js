import React from 'react'
import { shallow } from 'enzyme'
import AnswerBar from '.'

describe('containers:AnswerBar', () => {
  it('renders the AnswerBar', () => {
    expect(toJson(shallow(<AnswerBar />))).toMatchSnapshot()
  })
})
