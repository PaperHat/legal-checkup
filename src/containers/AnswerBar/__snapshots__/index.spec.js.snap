// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`containers:AnswerBar renders the AnswerBar 1`] = `
<StyledAnswerBar>
  <Connect(StyledResponseButton)
    type="Yes"
  />
  <Connect(StyledResponseButton)
    type="No"
  />
  <Connect(StyledResponseButton)
    type="Not sure"
  />
</StyledAnswerBar>
`;
